src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -233,13 +255,15 @@ export class ExamParticipationComponent implements OnInit, OnDestroy {
      *      --> in this case, we can even save all submissions with isSynced = true
      *
      * @param intervalSave is set to true, if the save was triggered from the interval timer
+     * @param force is set to true, when the current exercise should be saved (even if there are no changes)
      */
-    triggerSave(intervalSave: boolean) {
+    triggerSave(intervalSave: boolean, force: boolean) {
         // before the request, we would mark the submission as isSynced = true
         // right after the response - in case it was successful - we mark the submission as isSynced = false
         this.autoSaveTimer = 0;
 
-        if (this.currentSubmissionComponent?.hasUnsavedChanges()) {
+        if (force || this.currentSubmissionComponent?.hasUnsavedChanges()) {
```

Comment:
Suggestion: this.currentSubmissionComponent && force ||

Reply:
Why would you change the order?

Reply:
How can it happen that the currentSubmissionComponent is not available?